test/functionalTest/cases/3008_get_registration/get_reg_using_v2_created_using_v2.test

```diff
@@ -0,0 +1,167 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+GET registration using APIv2, created also using V2
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0 IPv4 -statCounters
+
+--SHELL--
+
+#
+# 01. Register entity E1/A1, using APIv2
```

Comment:
This .test seems to be pretty similar to v2_reg_create.test. Which is the difference?

Reply:
Look at the name of the tests:
```
get_reg_using_v2_created_using_v1.test
get_reg_using_v2_created_using_v2.test
```
v1/v2 used for registration prior to GET. We talked about this ...

Reply:
I'm not referring to the similarities between get_reg_using_v2_created_using_v1.test (btw, it is not part of this PR, is it?) and get_reg_using_v2_created_using_v2.test, but the similarities between v2_reg_create.test and get_reg_using_v2_created_using_v2.test

Reply:
ok, I understand.
Yes, it is possible that `get_reg_using_v2_created_using_v2.test` is part of some other test.
More than possible. I added it only because I had promised to ... :-)
To be removed

Reply:
Check before removal that it is actually duplicated. I may be wrong.

Reply:
Ah, ok.
Yes, it is possible that the test that I added (as promised) for issue 3008 is part of another test of issue 3004.

 And no, `get_reg_using_v2_created_using_v1.test` is from the PR of 'GET reg", and I had to use "create reg" in api v1, as `POST /v2/reg` was not yet implemented.

No problem in removing the test (get_reg_using_v2_created_using_v2.test)

Reply:
Fixed in 1a49367